src/Titon/Console/Console.hh

```diff
@@ -0,0 +1,27 @@
+<?hh // strict
+/**
+ * @copyright   2010-2015, The Titon Project
+ * @license     http://opensource.org/licenses/bsd-license.php
+ * @link        http://titon.io
+ */
+
+namespace Titon\Console;
+
+use Titon\Kernel\AbstractKernel;
+use Titon\Kernel\Middleware\Next;
+
+class Console extends AbstractKernel<Application, Input, Output> {
+
+    /**
+     * {@inheritdoc}
+     */
+    public function handle(Input $input, Output $output, Next<Input, Output> $next): Output {
```

Comment:
Does this function need to be added as an abstract method in the `AbstractKernel`? (I just did inheritdoc for now just in case).